packages/api/src/task/scheduler.ts

```diff
@@ -244,22 +246,24 @@ export default class TaskScheduler {
       ...updates,
     };
     const timestamp = task.status.updatedAt;
-    await this.queue.publishWebhook("events.task.status", {
+    await this.queue.publishWebhook("events.task.updated", {
       type: "webhook_event",
       id: uuid(),
       timestamp,
-      event: "task.status",
+      event: "task.updated",
       userId: task.userId,
       payload: {
         task: taskInfo(task),
       },
     });
     if (task.status.phase === "completed" || task.status.phase === "failed") {
-      await this.queue.publishWebhook("events.task.finished", {
+      let taskEvent: EventKey = `task.${task.status.phase}`; // after changing the webhook to task.completed
```

Comment:
The comment in the code was only a notice that the change would only make sense after changing the webhook to task.completed. Shouldn't include it here 😆 
```suggestion
      let taskEvent: EventKey = `task.${task.status.phase}`;
```